contracts/validator-registration-contract/validator_registration.sol

```diff
@@ -16,14 +15,15 @@ contract ValidatorRegistration {
     );
 
     uint public constant DEPOSIT_SIZE = 32 ether;
-    uint public constant DEPOSITS_FOR_CHAIN_START = 8;
+    uint public constant DEPOSITS_FOR_CHAIN_START = 8; // 2**14
```

Comment:
What does this comment mean? 2**14 is `16384`

Reply:
changed